plugin/src/main/java/org/opensearch/ml/plugin/MachineLearningPlugin.java

```diff
@@ -153,7 +145,15 @@
 import org.opensearch.threadpool.ThreadPool;
 import org.opensearch.watcher.ResourceWatcherService;
 
-import com.google.common.collect.ImmutableList;
+import java.util.Collection;
```

Comment:
Are these new imports necessary? If not, you can remove these

Reply:
These are not added by my changes. They were already there